tendrl/gluster_integration/sds_sync/georep_details.py

```diff
@@ -187,7 +187,7 @@ def aggregate_session_status():
                 sessions = []
             for session in sessions:
                 session_status = None
-                session_id = session.key.split("GeoRepSessions/")[-1]
+                session_id = session.value.split("GeoRepSessions/")[-1]
                 pairs = NS.gluster.objects.GeoReplicationPair(
```

Comment:
can we use session.session_id here?

Reply:
I checked but the field is not populated somehow after load_all